gradle/dependency-management/capabilities.json

```diff
@@ -102,5 +102,13 @@
         ],
         "selected": "cglib:cglib",
         "upgrade": "3.2.6"
+    },
+    {
+        "name": "jsch",
```

Comment:
This is useful to make sure we do not accidentally re-introduce the old `jsch` in the future. But on its own, it will not solve all issues as you discovered and commented.

Reply:
https://github.com/gradle/gradle/pull/20387#discussion_r861992460